format/FlightSql.proto

```diff
@@ -867,6 +867,69 @@ enum SqlSupportsConvert {
   SQL_CONVERT_VARCHAR = 19;
 }
 
+/*
+ * Represents a request to retrieve information about data type supported ona Flight SQL enabled backend.
+ * Used in the command member of FlightDescriptor for the following RPC calls:
+ *  - GetSchema: return the schema of the query.
+ *  - GetFlightInfo: execute the catalog metadata request.
+ *
+ * The returned schema will be:
+ * <
+ *   type_name: utf8 not null (The name of the data type, for example: VARCHAR, INTEGER, etc),
+ *   data_type: int not null (The SQL data type),
```

Comment:
To clarify, this is a database-specific type code uniquely identifying a data type (the "concise type" in ODBC terms)?

Reply:
Or is this actually database-specific? It seems ODBC has defined values for this.

Reply:
The data_type field is standard for ODBC and JDBC for the ANSI data types such as VARCHAR. This comment makes mentions of these being mapped from ANSI codes: https://github.com/aosm/iodbc/blob/d08b69dda97da494821a34ffb8aa56b4bac50b0f/iodbc/include/sql.h#L155 .

This can be a DBMS-specific type if it's a custom data type for the source.